subroom.lua

```diff
@@ -149,8 +149,8 @@ function ToArrival( tUser, sMessage )
 			Core.SendPmToUser( tUser, sTo, "The commands available are: help, list, join, invite, kick, police and leave" )
 			return true
 		end
-	elseif sCmd == "mod" or sCmd == "police" and ('01'):find( tostring(tUser.iProfile) ) then
-		local sNewMod = sData and sData:match( "^(%S+)" )
+	elseif (sCmd == "mod" or sCmd == "police") and tUser.bOperator then
```

Comment:
You can't successfully use `tUser.bOperator` here. The [documentation][1] is pretty clear about this:

> **Data available after `Core.GetUserAllData` or `Core.GetUserData` call:**

Use `Core.GetUserData(tUser, 11)`; based on the [data IDs][2] list.

_P.S._: Would you __please__ stop resetting history?

  [1]: https://hit-hi-fit-hai.github.io/PtokaX/user.html
  [2]: https://hit-hi-fit-hai.github.io/PtokaX/userdataids.html